src/.clang-format

```diff
@@ -1,11 +1,10 @@
----
 Language:        Cpp
```

Comment:
Since #2407 has been closed, I think this file needs to be taken off this PR?

Reply:
I am not sure about let umar respond on that.

Reply:
The changes can be reverted. Not the file

Reply:
Thanks for reverting the changes - for some reason I was thinking that the clang format file wasn't in master yet.